### DISCUSSION TITLE
How can I update palworld-server-docker's version?

### DISCUSSION THREAD

**Discussion by jexterliangsufe** [thijsvanloef/palworld-server-docker#310] [Q&A] [answered]
I started a server when the palworld-server-docker's version is 0.18.0. How can I update the version to the newest without destroying the existed server's files?

**Comment by thijsvanloef** [comment, marked as answer]
If you mounted your config using the volume mount from the example:
```yaml
      volumes:
         - ./palworld:/palworld/
```
Then you should be able to update your image using the following commands:

- `docker compose down --rmi all`
- `docker compose up -d`

**Reply by jexterliangsufe** [reply]
Thanks. I will try.
